test1/task2/main.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stddef.h>
+
+
+int selectionSort(int *array, int len)
+{
+    if (array == NULL)
+    {
+        return -1;
+    }
+    for (int i = 0; i < len; ++i)
```

Comment:
```suggestion
    for (int i = 0; i < len - 1; ++i)
```
Лишняя итерация в конце.